lib/formfields/customer/domains/formfield.domains_edit.php

```diff
@@ -165,6 +165,56 @@
 						)
 					)
 				)
+			),
+			'section_e' => array(
+				'title' => $lng['admin']['mailserversettings'],
+				'image' => 'icons/domain_edit.png',
+				'fields' => array(
+					'dkim' => array(
+						'visible' => (\Froxlor\Settings::Get('dkim.use_dkim') == '1' ? true : false),
+						'label' => 'DomainKeys',
```

Comment:
missing language string

Reply:
This was intended. There is no language dependent difference for a naming. It must be in all languages "DomainKeys" (or similar)...

Reply:
I hope I understand: you want something like "use DKIM"... I will implement.

Reply:
doesn't matter, please always use the language files